scripts/jenkins/cloud/ansible/group_vars/all/all.yml

```diff
@@ -40,6 +40,7 @@ sles_media_server: "ibs-mirror.prv.suse.net"
 rhel_enabled: "{{ rhel_computes is defined and rhel_computes | int > 0 }}"
 ses_enabled: False
 ses_rgw_enabled: False
+openid_connect_enabled: True
```

Comment:
Do you want to enable this by default for all deployments? We can enable this by default in the CI gating job only?

Reply:
The changes look good to me. I think that it should be do in all cases, since that will mean that this will be checked on incoming changes before they are merged and not just on the staging to devel gate.
Are there any update or upgrade implications fir this. I would not think there would be,  but we need to know for sure.

Reply:
Yes we want it to be enabled for all jobs.

I can't think of any upgrade issues as this is a new feature. No configuration overlap with existing features.

Reply:
agreed